File: app/src/main/java/com/example/notesholder/InfoFragment.java
package com.example.notesholder;

import android.os.Bundle;

import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

public class InfoFragment extends Fragment {

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {

            setHasOptionsMenu(true);
            ActionBar actionBar = ((AppCompatActivity)requireActivity()).getSupportActionBar();
            if (actionBar != null){
                actionBar.setSubtitle("Info");
            }


        return inflater.inflate(R.layout.fragment_info, container, false);
    }
}

File: app/src/main/java/com/example/notesholder/InfoFragment.kt
package com.example.notesholder

import com.example.notesholder.Utils.setSubtitleName
import android.view.LayoutInflater
import android.view.ViewGroup
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import com.example.notesholder.R

class InfoFragment : Fragment() {
    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        setHasOptionsMenu(true)
        val actionBar = (requireActivity() as AppCompatActivity).supportActionBar
        setSubtitleName(actionBar, "Info")
        return inflater.inflate(R.layout.fragment_info, container, false)
    }
}